core/src/main/java/org/eclipse/hono/util/CommandResult.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2016, 2018 Bosch Software Innovations GmbH.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *    Bosch Software Innovations GmbH - initial creation
+ */
+
+package org.eclipse.hono.util;
+
+import io.vertx.core.buffer.Buffer;
+
+/**
+ * A container for the result returned by Hono's registration API.
+ *
+ */
+public final class CommandResult extends RequestResponseResult<Buffer> {
```

Comment:
This doesn't seem to be specific to Command and Control at all. In fact, it couldn't be more generic, so why don't we call it `GenericResult`?

Reply:
Or BufferResult?